spec/features/admin/forms_spec.rb

```diff
@@ -1449,16 +1470,14 @@
           visit admin_form_path(form)
           fill_in "form_tag_list", with: "health benefits"
           page.find('#form_tag_list').native.send_keys :tab # to lose focus
+          expect(page).to have_content("health benefits".upcase)
```

Comment:
this is separate, and about tightening a flaky spec that's been impacting some builds